src/pages/RepoPage/ATSTab/ATSConfigured.tsx

```diff
@@ -0,0 +1,47 @@
+import A from 'ui/A'
+import Button from 'ui/Button'
+import Icon from 'ui/Icon'
+
+const ATSConfigured: React.FC = () => {
+  return (
+    <div className="flex flex-col justify-evenly space-y-7 bg-ds-gray-primary p-6">
+      <span className="flex flex-row space-x-2">
+        <div className="text-green-500">
+          <Icon name="checkCircle" />
+        </div>
+        <div>
+          <h1 className="mb-2 text-base font-semibold">
+            Automated Test Selection Configured
+          </h1>
+        </div>
+        <p className="flex max-h-4 rounded bg-ds-pink-tertiary px-2 text-xs font-semibold text-white">
+          BETA
+        </p>
+      </span>
+      <p>
+        {`\u{1F50A}`} <strong>Questions and feedback?</strong> Let us know your
+        thoughts&nbsp;
+        <A
+          to={{ pageName: 'feedback' }}
+          hook={undefined}
+          isExternal={undefined}
+        >
+          here
+        </A>
+        .
+      </p>
+      <div className="w-1/5">
+        <Button
```

Comment:
Buttons don't seem centered because they have the external icon in there in dark gray (hard to see). There's a prop, I forget the name, to hide the external icon, might be hideExternalIcon or something? But yeah happens here and in the other button as well